### PR TITLE
Ntp -dont merge yet

### PR DESCRIPTION
**Pull request by domenicbove**
# Description

start the ntp service on the hosts

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: roles/confluent.common/tasks/debian.yml [hunk 1]
```diff
@@ -28,3 +28,14 @@
 
 - import_tasks: debian_jvm.yml
   when: ansible_distribution == "Debian"
+
+- name: Install NTP
+  apt:
+    name: ntp
+    state: present
+
+- name: Start NTP Service
```
Comment (domenicbove) [inline roles/confluent.common/tasks/debian.yml]: this service most likely uses the internet, should make this configurable

File: roles/confluent.common/tasks/debian.yml [hunk 2]
```diff
@@ -28,3 +28,16 @@
...
+
+- name: Install NTP
+  apt:
+    name: ntp
+    state: present
+  when: ntp_enabled|bool
+
+- name: Start NTP Service
+  systemd:
+    name: ntp
```
Comment (stan-confluent) [inline roles/confluent.common/tasks/debian.yml]: on redhat the name of the service is ntpd, while the name of the package is ntp.
should it be ntpd or ntp on debian? just wanted to make sure its not a typo :)

Comment (domenicbove) [reply roles/confluent.common/tasks/debian.yml]: I found this out when testing on ubuntu and centos. idk why the service names are different

Comment (stan-confluent) [reply roles/confluent.common/tasks/debian.yml]: nice, thanks for confirming :) 
On the side note - ugh, why would they name the same thing differently?

**Comment by domenicbove** [comment]
There's a better way to handle time synchronization. right now we'll just make sure it documented. But forcing ntp on people is not the best

**Pull request closed without merging**